sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -280,7 +284,10 @@ public void execute(String portalName, int maxRows, ResultReceiver resultReceive
                     clearState();
                     ListenableFuture<?> result = portal.sync(planner, statsTables);
                     if (UNNAMED.equals(portal.name())) {
-                        portal.close();
+                        // The Postgres Wire Protocol states that unnamed portals are implicitly closed.
+                        // However if we would close the portal here, we would also interrupt the result receiving operations.
+                        // Instead we mark it as finished instead and close it with the next bind() call if necessary()
+                        portal.finish();
```

Comment:
My reasoning was that unless there have been `execute` calls `sync` is basically a no-op because there are no `pendingExecutions` - so this path shouldn't be hit.

But I'm not sure how a named portal may be re-used